docs/examples/provision-with-terraform.md

```diff
@@ -0,0 +1,220 @@
+
+This is achieved with the `liqo_generate` resource of the `liqo` Terraform provider instance, configured with either the config file or the full list of parameters of the remote cluster:
+
+```terraform
+provider "liqo" {
+  alias = "milan"
+  kubernetes = {
+    config_path = kind_cluster.milan.kubeconfig_path
+  }
+}
+
+resource "liqo_generate" "generate" {
+
+  depends_on = [
+    helm_release.install_liqo_milan
+  ]
+
+  provider = liqo.milan
+
+}
+```
+
+### Run peering procedure
+
+Once the `generate_resource` is created, Terraform will continue with the [out-of-band peering](/features/peering.md) procedure leveraging the output parameters of the previous resource.
+
+This is achieved with the `liqo_peering` resource of the `liqo` Terraform provider instance, configured with either the config file or the full list of parameters of the local cluster:
+
+```terraform
+provider "liqo" {
+  alias = "rome"
+  kubernetes = {
+    config_path = kind_cluster.rome.kubeconfig_path
+  }
+}
+
+resource "liqo_peering" "peering" {
+
+  depends_on = [
+    helm_release.install_liqo_rome
+  ]
+
+  provider = liqo.rome
+
+  cluster_id      = liqo_generate.generate.cluster_id
+  cluster_name    = liqo_generate.generate.cluster_name
+  cluster_authurl = liqo_generate.generate.auth_ep
+  cluster_token   = liqo_generate.generate.local_token
+
+}
+```
+
+You can check the peering status by running:
+
+```bash
+kubectl get foreignclusters
+```
+
+The output should look like the following, indicating that the cross-cluster network tunnel has been established, and an outgoing peering is currently active (i.e., the *Rome* cluster can offload workloads to the *Milan* one, but not vice versa):
+
+```text
+NAME    TYPE        OUTGOING PEERING   INCOMING PEERING   NETWORKING    AUTHENTICATION   AGE
+milan   OutOfBand   Established        None               Established   Established      12s
+```
+
+At the same time, you should see a virtual node (`liqo-milan`) in addition to your physical nodes:
+
+```bash
+kubectl get nodes
+```
+
+```text
+NAME                 STATUS   ROLES                  AGE     VERSION
+liqo-milan           Ready    agent                  14s     v1.23.6
+rome-control-plane   Ready    control-plane,master   7m56s   v1.23.6
+rome-worker          Ready    <none>                 7m25s   v1.23.6
+```
+
+### Offload a namespace
+
+If you want to deploy an application that is scheduled on a Liqo virtual node (hence, it is offloaded on a remote cluster), you should first create a namespace where your pod will be started.
+
+This can be achieved with `kubernetes_namespace` of [kubernetes](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs) Terraform provider, configured with either the config file or the full list of parameters of the local cluster.
+
+Then tell Liqo to make this namespace eligible for the pod offloading.
+
```

Comment:
I would personally drop this empty line (to join the two sentences, which are related).